### ISSUE TITLE
Cant install on windows

### ISSUE THREAD

**Issue by mc17336** [htm-community/htm.core#943]
Hi,

I tried installing using the read me, using python -m pip install -i https://test.pypi.org/simple/ htm.core but got the following error:
ERROR: Could not find a version that satisfies the requirement htm.core (from versions: none)
ERROR: No matching distribution found for htm.core

I then tried building from the setup.py doc but get the following error:
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.1520.0_x64__qbz5n2kfra8p0\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

I've installed requirements using the requirements.txt file and also ensured I have cmake installed. Can anyone help please?

Thanks, M

**Comment by dkeeney** [comment]
The pip install only works with python 3.7 and maybe 3.8.  So you will need to build from sources.  See ReadMe.

Would anyone like to take over the generation of the PyPi files?

**Comment by HMaker** [comment]
@dkeeney why it's hosted on test pypi and not on main pypi repo?

**Comment by dkeeney** [comment]
> @dkeeney why it's hosted on test pypi and not on main pypi repo?

Because we never really finished transitioning from a proof-of-concept trial to full production into PyPi.  So, with the exception of the few combinations of Python versions and OS versions that we exported, I suggest that everyone build from sources.

Having said that, I realize that having a complete set of PyPi modules would certainly be useful for the Python developers out there because they would not be required to have a C++ compiler installed.  What we need is someone willing to take on the project of doing all of the builds or setting up a build stream that can build them.

**Comment by ctrl-z-9000-times** [comment]
I can take a look into publishing htm-core on pypi, not now but later this week.
I've published other packages on pypi before, so I'm familiar with the process.
I think it should be possible to set up github to automatically do a lot of the work. (https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/)
Also, IIRC Breznak did the last release so he might know something about the process?

**Comment by ctrl-z-9000-times** [comment]
Update: I don't think I will be getting around to fixing this anytime soon, so if someone else wants to drive this issue it's all yours.

Status:
- Blocked by: #950
- Would be nice to have: #952 
- Another issue: We think that Matt (rest in piece) created the pypi account. Someone should talk to everyone at Numenta to see if they inherited the password. Otherwise we will need to make a new pypi account. PyPi accounts are free, but we will need to decide how to manage the credentials for any new account.

**Comment by dkeeney** [comment]
@ctrl-z-9000-times  Thanks for doing as much as you did.
I cannot work on this right now but perhaps latter.
